bindings.cpp

```diff
@@ -3635,24 +3438,9 @@ bool DeRestPluginPrivate::checkSensorBindingsForClientClusters(Sensor *sensor)
     //quint8 srcEndpoint = sensor->fingerPrint().endpoint;
     std::vector<quint16> clusters;
 
-    if (sensor->modelId().startsWith(QLatin1String("RWL02")) || // Hue dimmer switch
-        sensor->modelId().startsWith(QLatin1String("ROM00")) || // Hue smart button
-        sensor->modelId().startsWith(QLatin1String("ElkoDimmer")) || // Elko dimmer
-        sensor->modelId().startsWith(QLatin1String("E1E-"))) // Sengled smart light switch
```

Comment:
Do we have "ElkoDimmer" and "E1E-" already on DDF?

Reply:
Damn, I completely missed those.  I think we’d better keep the code for these.